lib/web.ts

```diff
@@ -43,12 +45,12 @@ export default class AutoConsent {
         selfTest: null,
     };
     domActions: DomActions;
-    filtersEngine: FiltersEngine;
-    protected sendContentMessage: MessageSender;
-    protected cosmeticStyleSheet: CSSStyleSheet;
-    protected focusedElement: HTMLElement = null;
+    filtersEngine: FiltersEngine | null = null;
+    sendContentMessage: MessageSender;
+    protected cosmeticStyleSheet?: CSSStyleSheet;
+    protected focusedElement: HTMLElement | null = null;
```

Comment:
Not a fan of making all of the fields optional or `| null`. It’s much harder to reason about types that are too permissive and it’s a poor developer experience to constantly have to do null checks.

Reply:
But these can actually be nullish depending on the site and config. I made them all optional so we have only one actual nullish value